### PR TITLE
Tskit for population geneticists

### PR DESCRIPTION
**Pull request by hyanwong**
Next round of addressing #191

### CODE REVIEW

File: popgen.md [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+populations in the model. The DNA sequences and their ancestry are stored in a succinct
+tree sequence named `ts`:
+
+```{code-cell}
+contig = species.get_contig("chr1", mutation_rate=model.mutation_rate, right=20_000)
+samples = {"AFR": 4, "EUR": 4, "ASIA": 4, "ADMIX": 4} # 16 diploid samples
+engine = stdpopsim.get_engine("msprime")
+ts = engine.simulate(model, contig, samples, seed=9)
+print(f"Simulated a tree sequence of {ts.num_samples} haploid genomes:")
+print(f"{ts.num_sites} variable sites over {ts.sequence_length/1e3} kb of sequence")
```
Comment (benjeffery) [inline popgen.md:130]: Is `kb` the standard for kilobases? My computer brain reads it as kilobytes.

File: popgen.md [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+]
+
+labels = { # Label samples by population
+    u: ts.population(ts.node(u).population).metadata["name"].capitalize()
+    for u in ts.samples()
+}
+
+tree.draw_svg(
+    size=(800, 500),
+    # canvas_size=(800, 520),  # Make room for labels - only valid in tskit >= 0.5.4
```
Comment (benjeffery) [inline popgen.md:210]: Let's file an issue to update this when 0.5.4 is released.

**Review by benjeffery — APPROVED** [review]
This is excellent @hyanwong, thanks!
Just a couple of small comments.

File: popgen.md (hunk 1 above)
Comment (hyanwong) [reply popgen.md]: It is pretty standard, but easy enough to spell it out in full, so I've done so in the latest push

File: popgen.md (hunk 2 above)
Comment (hyanwong) [reply popgen.md:210]: Good point. https://github.com/tskit-dev/tutorials/issues/211

**Pull request merged**